Content.Shared/Maps/TurfHelpers.cs

```diff
@@ -194,7 +194,7 @@ private static Box2 GetWorldTileBox(TileRef turf)
         {
             var map = IoCManager.Resolve<IMapManager>();
             var tileGrid = map.GetGrid(turf.GridIndex);
-            var tileBox = Box2.UnitCentered.Scale(tileGrid.TileSize);
+            var tileBox = Box2.UnitCentered.Scale(tileGrid.TileSize).Scale(0.9f);
```

Comment:
Maybe add a comment to explain why you added `Scale(0.9f)`?

Reply:
kay